NetSuiteProject/Exercise.js

```diff
@@ -2,22 +2,65 @@
 function add(x,y){
 	return x + y;
 };
+
+function addition(x,y){
+	if(typeof add (x,y) === "string"){
+	alert("This function only accepts Integers")
+	}
+	else {return x + y;}
```

Comment:
In all of your new functions, you've duplicated the logic of the original functions. For example here, `add` already does `return x+y;`, but so does `addition`. If you are keeping the two-function approach for handling invalid inputs, can you think of a way to do that without duplicating the business logic of `add` and the other original functions?